### PR TITLE
Create between-two-cities.md

### PR DESCRIPTION
**Pull request by rigard**
I am in the process of writing up summarised versions of many games - the idea being for use as refreshers or teaching aids. I understand this is not exactly what your site is doing, but take a look and maybe there is a place for it - differently tagged, or just as it is. If not, feel free to ignore.

Contents and tags are stubbed for now.

This game is Between Two Cities: https://boardgamegeek.com/boardgame/168435/between-two-cities

### CODE REVIEW

**Comment by camden** [comment]
@rigard Thanks for the PR! This kind of content is definitely welcome here - even a summary is better than nothing. As for the content & tags, is that something that you want to discuss, or is that just a note for yourself to finish at a later time?

**Comment by rigard** [comment]
Hey. Ok, great - happy to contribute more. I do think they should be identified as summaries, as a person really needs to have read the rules previously or played the game already, but I can just tag them with "summary". I'll fill out the stubs.

**Comment by camden** [comment]
Thanks @rigard! I’ll give this a final once over then merge this when I get home tonight.

**Comment by rigard** [comment]
I see you merged "Snow Tails". This one OK for you?

**Comment by camden** [comment]
Hey, yeah I added a "review" about that last bit, but I hadn't finalized it until just now.

File: rulebooks/between-two-cities.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+**Offices:** Score for number of tiles, plus bonus next to tavern
+
+**Parks:** Increase score for larger adjacent groups (can have multiple groups)
+
+**Houses:** Score according to diversity across city (different types of other tiles)
+
+# Notes
+
+* Quadropolis
```
[the remark this answers is not in the document]
Comment (rigard) [reply rulebooks/between-two-cities.md]: Hmm... It was added for my personal purposes. It's basically a note that it has similarities to the game Quadropolis. Happy to remove that, or happy if you remove that - it's not a requirement for anything.

[the remark this answers is not in the document]
Comment (camden) [reply rulebooks/between-two-cities.md]: Hey, yeah let's remove this, then I'll merge it in. :)

**Comment by camden** [comment]
Thanks @rigard! Sorry for the hassle. :)

**Pull request merged**